test/cpp_api_parity/sample_module.py

```diff
@@ -0,0 +1,107 @@
+import torch
+from cpp_api_parity import has_parity
+
+'''
+`SampleModule` is used by `test_cpp_api_parity.py` to test that Python / C++ API
+parity test harness works for `torch.nn.Module` subclasses.
+
+When `SampleModule.has_parity` is true, behavior of `reset_parameters` / `forward` /
```

Comment:
`has_parity` isn't an attribute on SampleModule anymore so this should be changed

Reply:
`has_parity` context manager is removed and `has_parity` remains an attribute of `SampleModule`